.github/workflows/reusable-agent-build-container-images.yml

```diff
@@ -15,17 +15,24 @@ on:
         description: "AWS region for remote builds."
         type: string
 
-      aws_private_key_name:
-        description: "Name of a private key that is used to connect to AWS EC2 instances that will be created by this workflow."
-        type: string
-
       cicd_workflow:
         description: "Additional string that has to differentiate AWS object from another workflows."
         type: string
+      
+      builder_name:
+        type: string
+
+      base_image:
+        type: string
+
+      architectures:
```

Comment:
If the is a fixed list of values we support for this parameter, we should perhaps utilize `options` attribute (https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/).

Reply:
I am sending there a json representing a list.
There's no structure for that in GHA inputs unfortunatelly.

https://github.com/scalyr/scalyr-agent-2/blob/parametrize_gha_2/.github/workflows/README.md

Reply:
For workflow_call you can use only boolean, number and string.

Reply:
Ah, OK, so the `choice` type  is not supported for `workflow_call` trigger. Good to know, thanks.

For reference:

* https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputsinput_idtype
* https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype